src/rtcSessionHelpers.ts

```diff
@@ -43,19 +44,28 @@ async function makePreferredLivekitFoci(
     preferredFoci.push(focusInUse);
   }
 
-  // Prioritize the client well known over the configured sfu.
-  const wellKnownFoci =
-    rtcSession.room.client.getClientWellKnown()?.[FOCI_WK_KEY];
-  if (Array.isArray(wellKnownFoci)) {
-    preferredFoci.push(
-      ...wellKnownFoci
-        .filter((f) => !!f)
-        .filter(isLivekitFocusConfig)
-        .map((wellKnownFocus) => {
-          logger.log("Adding livekit focus from well known: ", wellKnownFocus);
-          return { ...wellKnownFocus, livekit_alias: livekitAlias };
-        }),
-    );
+  // Prioritize the .well-known/matrix/client, if available, over the configured SFU
+  const domain = rtcSession.room.client.getDomain();
+  if (domain) {
+    // we use AutoDiscovery instead of relying on the MatrixClient having already
+    // been fully configured and started
+    const wellKnownFoci = (await AutoDiscovery.getRawClientConfig(domain))?.[
+      FOCI_WK_KEY
+    ];
```

Comment:
Maybe we want to do a check if the client already knows about the well known? so we save the http request in that case. I suspect this is a race condition since it worked at some point.
So maybe at some point this works again.

I also think however that we might want to get to the root of where this race condition occurs. And when we find it, we can use the old logic again.
```suggestion
    const wellKnownFoci = rtcSession.room.client.getClientWellKnown()?.[FOCI_WK_KEY] ? (await AutoDiscovery.getRawClientConfig(domain))?.[
      FOCI_WK_KEY
    ];
```